### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by runeflobakk** [unruly/java-8-matchers#21]
Hello,

I was wondering if this project is maintained, as there seems to not be that much activity going on. There are some old issues, and a couple of pull-requests just lying around.

I think this is a very useful library, and I actively use it on a daily basis, so I am wondering about what are the possibilities of me to become a maintainer of this repository, as opposed to be working on my own fork and send pull-requests? The last merged pull-request was mine, and approved by @mrwilson , so I am shamelessly mentioning you to maybe provide som insights on this matter :)

For reference, this is my previous contributions to this library:
- #20 Add missing space in mismatch description (fixes #15)
- #17 Documentation with example code for Java8Matchers (as a response to unruly/java-8-matchers: Issue #9)
- #11 Support building matchers with lambdas (fixes #9)
- #4 StreamMatchers.contains(..) handles nulls (fixes #3)

**Comment by CliveEvans** [comment]
It's a good question. I certainly still use it, but no longer have the keys to do much to it, since I left. Tagging @bruceeddy and @jahed as other people who might be able to answer ...

**Comment by jahed** [comment]
I left Unruly too so I'm no longer in a position to transfer ownership.

What I can say is that the package is tied to `co.unruly` so that is likely a problem in terms of authenticating releases and general company branding.

If no one at Unruly is maintaining this repo, maybe it's best and easiest if @runeflobakk forked the project and released under another domain/namespace. That seems to be the common approach with Java libraries.

**Comment by bruceeddy** [comment]
Hi All,

Good to hear the project is still of interest.  It's still in use here at
Unruly, and it's encouraging to hear that others are interested too.

I'm currently travelling, and I can only see Clives's and Jahed's
responses, not the issue itself, but I assume that the topic is a PR which
hasn't been merged yet.  I will be happy to arrange for merges to happen,
and/or a new release into maven central.  It's certainly true that there
hasn't been much activity on the project for a few years, so it's overdue
some maintenance if people are still using it.

--bruce

**Comment by runeflobakk** [comment]
The currently open pull-requests are just two small ones.
But it is good to hear that there is interest in further maintaining it. I am hoping I will find some time to fix the small amount of currently registered issues (if no one beats me to it 😉), and I also want to do some housekeeping chores to Maven pom-file, and such. I also have som ideas for enhancing the `Java8Matchers.where` variants. I'll just ping you when I have something ready 👍

**Comment by runeflobakk** [comment]
@bruceeddy If possible, I would appreciate if #20 , #23 and #22 (the last one includes #19 as well) could be reviewed and merged. :)

**Comment by runeflobakk** [comment]
@bruceeddy It would be really nice to have the mentioned pull-requests merged before starting on the remaining registered issues.

**Comment by runeflobakk** [comment]
@bruceeddy Sorry to nag, but are there any new info regarding https://github.com/unruly/java-8-matchers/pull/22#issuecomment-627858039 ?

**Comment by mrwilson** [comment]
@runeflobakk Hi, I'm one of the original contributors and I've hard-forked (and released) this repo at https://github.com/mrwilson/java-8-matchers - the current version on Central is now 1.8, which is the same as the original 1.6

If you raise your changes as PRs against the fork (the package is now `uk.co.probablyfine.matchers` rather than `co.unruly.matchers`) I will aim to merge and release them soon.

**Comment by runeflobakk** [comment]
@mrwilson Great! I'll get around to this soon, as I am on vacation at the moment 👍
